script/script.js

```diff
@@ -211,3 +211,17 @@ document.addEventListener('click', (event) => {
     }
   });
 });
+
```

Comment:
- [ optional] great job with the form. it validates perfectly.  kindly fix the brightness of your form text input
![Screenshot (176)_LI](https://user-images.githubusercontent.com/65298048/175340120-b6008a61-cf1a-4941-9dd1-f09e98ce472c.jpg)